### PR TITLE
Choropleth

### PR DESCRIPTION
**Pull request by maharit108**
Not to merge.......

### CODE REVIEW

**Comment by maharit108** [comment]
currently works only for county layer.... work needed on zipcode layer

File: src/mapbox/Colors.js [hunk 1]
```diff
@@ -2,7 +2,19 @@
...
  */
 export const highlight = "#fff";
-export const countyFill = "#059713";
+export const countyFill = "#fff";
 export const countyOutline = "#124c1b";
 export const zipcodeFill = "#F3AD1C";
-export const zipcodeOutline = "#a4720b";
\ No newline at end of file
+export const zipcodeOutline = "#a4720b";
+export const colorMapFill = [
```
Comment (AnansiOmega) [inline src/mapbox/Colors.js:9]: Would an array of objects be a better way to store the colors. For example: { 'green': '#00FF00' }

**Review by arosenfeld2003 — COMMENTED** [review]
@maharit108 great start!  I think we should go ahead and push this commit as a jumping off point for rendering demographic color layers on the map.

File: src/mapbox/CountyLayer.js [hunk 2]
```diff
@@ -1,7 +1,9 @@
-import React from 'react'
+import React, { useMemo } from 'react'
```
Comment (arosenfeld2003) [inline src/mapbox/CountyLayer.js:1]: Does it make sense to move this logic into a separate component e.g. CountySnapshotOverlay... or maybe even combine it with the CountySnapshotOverlay?

Comment (arosenfeld2003) [inline src/mapbox/CountyLayer.js:1]: I like the use of `useMemo` here... makes sense

**Pull request closed without merging**